src/domain/operator.js

```diff
@@ -0,0 +1,23 @@
+
+const operator = {
+    EQUAL: {
+        name: 'EQUAL',
+        execute: (entity, value) => {
+            return entity === value
+        }
+    },
+    EXIST: {
+        name: 'EXIST',
+        execute: (entity,value) => {
```

Comment:
Fijate que para poder tener polimorfismo en los operadores, tuviste que llegar a esto. Tenés un metodo que recibe un entity y un value y no hace nada con el value. 
Incluso te la pongo a prueba mas:
El operador IN, que chequea si un valor está en una lista de valores, como lo harías?
Pensá como podría hacer para que mantener el polimorfismo del execute pero independiente de que valores use el operador?